NotebookAnalyzers/pynblint.py

```diff
@@ -351,3 +305,31 @@ def count_raw_cells(nb_dict):
         if cell["cell_type"] == 'raw':
             counter = counter + 1
     return counter
+
+
+def get_bottom_md_lines_ratio(nb_dict, bottom_size=4):
+    """
+        Percentage of markdown rows in the last cells of the notebook out of the total of md rows
+
+        Args: nb_dict(dict): python dictionary object representing the jupyter notebook.
+
+        Returns: md_bottom_cells/md_first_cells: Percentage of markdown rows in the last cells of the notebook
```

Comment:
Come specificato nell'altra PR, anche in questo caso specificherei nella documentazione l'assunzione di fondo:

```python
if bottom_size < total_cells / 3:
```